### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by sdgsa** [snigi-gupta/AttendingUniversityAtBuffalo#47] [General]
1. Will a thin blanket do the job or we need comforters? Price?
2. Do we need filter for drinking water? Price?
3. How to transfer luggage from JFK terminal 5 to 4? 
4. Is there any wifi unlimited plan? Price?

**Comment by snigi-gupta** [comment]
Hi @sdgsa,

1. No, a thin blanket won't usually do. Winters are tough in Buffalo. I bought the [Mainstay's comforter](https://www.walmart.com/search/?query=mainstays%20comforter&typeahead=mainstays%20com) from [Walmart](https://www.walmart.com/), and it has been solving my purpose.
2. I would suggest getting a [Brita filter](https://www.amazon.com/Brita-UltraMax-Dispenser-Longlast-Filter/dp/B079DDN98K/ref=sr_1_6?dchild=1&keywords=brita&qid=1626045368&sr=8-6). The tap water has fluorine in it. And I personally do not like the taste. When I use a Brita filter, it works for me. This is a personal choice though. Tap water is perfectly drinking water.
3. They must be connected and you can buy a trolley (in USA the lugagge carts/trolley are paid, $3-$6 I think) and transfer to the terminal. I would recommend, asking the information center or the airline attendee.
4. I used Spectrum and Verizon both. Spectrum started causing some issues in year 2 so I chose Verizon. Spectrum is cheaper and it was super good for the first year. I was paying $49.something(200 Mpbs). The second-year I moved to Verizon and was paying $51.30 (300Mbps).
